int32 batch_size
---
string[] image_path
geometry_msgs/TwistStamped[] commands


